driver/docker/docker.go

```diff
@@ -55,11 +55,20 @@ func (d *Driver) Config() map[string]string {
 		"PULL_ALWAYS":         "Always pull image, even if locally available (0|1)",
 		"DOCKER_DRIVER_QUIET": "Make the Docker driver quiet (only print container stdout/stderr)",
 		"OUTPUTS_MOUNT_PATH":  "Absolute path to where Docker driver can create temporary directories to bundle outputs. Defaults to temp dir.",
+		"CLEANUP_CONTAINERS":  "If true, the docker container will be destroyed when it finishes running. If false, it will not be destroyed. The supported values are true and false. Defaults to true.",
```

Comment:
Maybe we should invert the name and the logic, with something like "KEEP_CONTAINERS" ? So by default it's `false` and I guess the code in `SetConfig` would be simpler?

Reply:
Can you open a follow up issue for that Silvin?

Reply:
Done here #141